code/datums/station_traits/neutral_traits.dm

```diff
@@ -0,0 +1,65 @@
+/datum/station_trait/bananium_shipment
+	name = "Bananium Shipment"
+	trait_type = STATION_TRAIT_NEUTRAL
+	weight = 5
+	report_message = "Rumors has it that the clown planet has been sending support packages to clowns in this system"
+	trait_to_give = STATION_TRAIT_BANANIUM_SHIPMENTS
+
+/datum/station_trait/unnatural_atmosphere
+	name = "Unnatural atmospherical properties"
+	trait_type = STATION_TRAIT_NEUTRAL
+	weight = 5
+	show_in_report = TRUE
+	report_message = "System's local planet has irregular atmospherical properties"
+	trait_to_give = STATION_TRAIT_UNNATURAL_ATMOSPHERE
+
+/datum/station_trait/unique_ai
+	name = "Unique AI"
+	trait_type = STATION_TRAIT_NEUTRAL
+	weight = 5
+	show_in_report = TRUE
+	report_message = "For experimental purposes, this station AI might show divergence from default lawset. Do not meddle with this experiment."
+	trait_to_give = STATION_TRAIT_UNIQUE_AI
+
+/datum/station_trait/ian_adventure
+	name = "Ian's Adventure"
+	trait_type = STATION_TRAIT_NEUTRAL
+	weight = 5
+	show_in_report = FALSE
+	report_message = "Ian has gone exploring somewhere in the station."
+
+/datum/station_trait/ian_adventure/on_round_start()
+	for(var/mob in GLOB.mob_list)
+		if(!(istype(mob, /mob/living/simple_animal/pet/dog/corgi/ian) || istype(mob, /mob/living/simple_animal/pet/dog/corgi/puppy/ian)))
+			continue
```

Comment:
for(type in list) is faster here, as prophesied by giacom. even if you just wanna check for ian at the end, that's a significantly smaller list to iterate over